scripts/timing.py

```diff
@@ -149,6 +156,26 @@ def call_or_error(cmd: List[str]):
         raise Exception('Command {} exited with exit code {}'.format(cmd, return_code))
 
 
+def get_k8s_env() -> K8sEnv:
+    """Get current Kubernetes env. (or throw an exception)."""
+    p = subprocess.Popen(['kubectl', 'config', 'view', '-o', 'template', '--template={{ index . "current-context" }}'], stdout=subprocess.PIPE)
```

Comment:
are you aware of 'kubectl config current-context'?

this seems...overly complicated

Reply:
nnnnno. again, stack overflow strugglebus. 😳